dist/FindBin/lib/FindBin.pm

```diff
@@ -107,8 +109,8 @@ sub cwd2 {
 
 sub init
 {
- *Dir = \$Bin;
- *RealDir = \$RealBin;
+ *Dir = *Bin;
```

Comment:
What's the reason for this change?

Reply:
For clarity since glob assignment is black magic stuff, `*foo = \$bar` aliases the SCALAR slot of `*bar` to the SCALAR slot of `*foo`, whereas `*foo = *bar` aliases *all* slots - ARRAY, HASH, CODE, IO, etc. Only the scalar slot is used by this module so the existing code should be fine.

Reply:
You're suggesting it would be cheaper as an RV. Probably true.

Reply:
This is an alias, not a reference.